packages/in_app_purchase/in_app_purchase_android/android/src/test/java/io/flutter/plugins/inapppurchase/MethodCallHandlerTest.java

```diff
@@ -417,6 +417,7 @@ public void launchBillingFlow_ok_AccountId() {
   }
 
   @Test
+  @SuppressWarnings(value = "deprecation")
```

Comment:
Could you annotate these with the TODO comment linking to the issue, so we know why they are here and when we can clean them up?

Reply:
Added!